### PR TITLE
Added recovery beneficiaries

### PR DESCRIPTION
**Pull request by bioan**

### CODE REVIEW

File: Anabi.DataAccess.Ef/AnabiContext.cs [hunk 1]
```diff
@@ -40,6 +40,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             (new AssetsStorageSpacesConfig()).SetupEntity(modelBuilder);
             base.OnModelCreating(modelBuilder);
+            (new RecoveryBeneficiaryConfig()).SetupEntity(modelBuilder);
+            base.OnModelCreating(modelBuilder);
```
Comment (radu-ungureanu) [inline Anabi.DataAccess.Ef/AnabiContext.cs]: OnModelCreating is already called on the line above. I don't think you should call it twice

File: Anabi/Controllers/RecoveryBeneficiariesController.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        };
+
+        // GET: api/RecoveryBeneficiaries
+        [HttpGet]
+        public async Task<IEnumerable<RecoveryBeneficiary>> Get()
+        {
+            try
+            {
+
+                return await repository.GetAll().Select(selector).ToListAsync();
```
Comment (radu-ungureanu) [inline Anabi/Controllers/RecoveryBeneficiariesController.cs]: this try catch is redundant.

File: Anabi/Controllers/RecoveryBeneficiariesController.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            catch (Exception ex)
+            {
+
+                throw;
+            }
+        }
+
+        // POST: api/RecoveryBeneficiaries
+        [HttpPost]
+        public void Post([FromBody]string value)
```
Comment (radu-ungureanu) [inline Anabi/Controllers/RecoveryBeneficiariesController.cs]: I don't think we should add methods that are not implemented. if we need functionality in a later task, the developer will add it.

File: Anabi.DataAccess.Ef/AnabiContext.cs (hunk 1 above)
Comment (bioan) [inline Anabi.DataAccess.Ef/AnabiContext.cs]: Good catch.. no clue how that happened.

**Comment by bioan** [comment]
Updated

**Pull request merged**